src/briefcase/commands/run.py

```diff
@@ -248,6 +248,7 @@ def __call__(
         no_update: bool = False,
         test_mode: bool = False,
         passthrough: list[str] | None = None,
+        config_overrides: list[str] | None = None,
```

Comment:
needs to be passed to `finalize()`?

Reply:
Yes, but not any more with the revised approach that I've implemented.